core-meetings/2019-11-oss-meeting.md

```diff
@@ -0,0 +1,33 @@
+# Open Source Maintenance: November 2018 Core Meeting
+
+As stewards of React Native engineers worldwide, we've contemplated ways to improve alignment and visibility of the framework's direction, where community members and Facebook alike joined to discuss issues and propose responses.
+
+The community team is made up of volunteers who help maintain React Native, contribute to education materials, and improve the React Native developer's environment.
+
+## The core discussion
+
+This time, many members from both the community and the FB team met to talk about the Open Source repository and its maintenance.
+
+The first part of the conversation was about two aspects of the current release (0.57.x):
+
+- how the React sync commits land in master, and have impacted the releases
+- how to tackle the patch releases and cherry picks request, given that over time this is more complex and prone to merge conflicts (because of the speed at which master branch moves)
+
+During the discussion, the FB team also clarified a couple of points from their latest blogpost about [Open Source Roadmap](http://facebook.github.io/react-native/blog/2018/11/01/oss-roadmap), here reported for transparency sake:
+
+- the internal FB team is _not_ migrating to React Native Open Source, they are simply changing _how_ they consume the React Native APIs. Which means that, internally, instead of writing `const View = require('View')` they will be writing `const {View} = require('react-native')`
+
+- the FB team is not foreseeing taking management or ownership of the release cycle, commending the community for how they have handled it so far
```

Comment:
I think we should also talk about why this was decided. Some of the reasons were 

1. We would like more community participation, and would like to community to be able to control when the releases happen
2. The community probably know more about this than FB. The releases is how the community consumes RN, and should have the power to decide how they go out.

Reply:
We should also call out that we need help here. We plan to write down the release steps, and see how we can automate most of this.

Reply:
Ok, I'll word it better.